README.md

```diff
@@ -1,19 +1,19 @@
 [![][ButlerImage]][website] 
 
 # About
-In a nutshell, Jenkins CI is the leading open-source continuous integration server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
+In a nutshell, Jenkins is the leading open-source automation server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
```

Comment:
Done. After reading the wiki, I felt this was still to software delivery related when we were agreeing here on an even more generic automation application.